packages/core/core-flows/src/cart/steps/get-line-item-actions.ts

```diff
@@ -48,15 +48,18 @@ export const getLineItemActionsStep = createStep(
       if (existingItem && metadataMatches) {
         const quantity = MathBN.sum(
           existingItem.quantity as number,
-          item.quantity || 1
-        ).toNumber()
+          item.quantity ?? 1
+        )
 
         itemsToUpdate.push({
           selector: { id: existingItem.id },
           data: {
             id: existingItem.id,
             quantity: quantity,
             variant_id: item.variant_id!,
+            unit_price: item.unit_price || existingItem.unit_price,
```

Comment:
Q: should we use nullishcoalescing as well?

Reply:
converted them all. Lets see if that passes the specs. 🥂

Reply:
Looks good! all 🟢